### PR TITLE
remove unpack workflow from MMPK samples

### PR DESCRIPTION
**Pull request by JonLavi**
Hi both,
This is a quick PR to remove the 'unpack' workflow from this sample. 
With U7 there will be no more need for 'unpack' since all data types will support direct read. 
This is the only sample I'm aware of that uses `directreadsupport` or `unpack`.

Updated the README slightly to reflect the changes and added a 'how to use the sample' section.

Let me know what you think.

Thanks

### CODE REVIEW

File: scene/open-mobile-scene-package/README.md [hunk 1]
```diff
@@ -8,22 +8,20 @@ Open and display a scene from an offline Mobile Scene Package (.mspk).
...
 ## How it works
 
-1. Use the static method `MobileScenePackage.isDirectReadSupportedAsync(mspkData)` to check whether the package can be read in the archived form (.mspk) or whether it needs to be unpacked.
-2. If direct read is supported, use `isDirectReadSupported.get()` and instantiate a `MobileScenePackage` with the path to the .mspk file.
-3. If the mobile scene package requires unpacking, use `MobileScenePackage.unpackAsync(mspkPath, pathToUnpackTo)` and instantiate a `MobileScenePackage` with the path to the unpacked .mspk file.
-4. Call `mobileScenePackage.loadAsync` to load the mobile scene package. When finished, get the `ArcGISScene` objects inside with `mobileScenePackage.getScenes()`.
-5. Set the first scene in the object collection on the scene view with `sceneView.setArcGISScene(scene)`.
+1. Create a `MobileScenePackage` with the path to a .mspk file.
+2. Call `mobileScenePackage.loadAsync` to load the mobile scene package. When finished, get the `ArcGISScene` objects inside with `mobileScenePackage.getScenes()`.
+3. Set the first scene in the object collection on the scene view with `sceneView.setArcGISScene(scene)`.
```
Comment (tschie) [inline scene/open-mobile-scene-package/README.md]: Step 3 can just be "Set the first scene to the scene view."

File: scene/open-mobile-scene-package/src/main/java/com/esri/samples/open_mobile_scene_package/OpenMobileScenePackageSample.java [hunk 2]
```diff
@@ -53,56 +49,15 @@ public void start(Stage stage) throws Exception {
     // create a new scene view
     sceneView = new SceneView();
 
-    // load a mobile scene package
+    // create a mobile scene package from a file
     final String mspkPath = new File("./samples-data/mspk/philadelphia.mspk").getAbsolutePath();
```
Comment (tschie) [inline scene/open-mobile-scene-package/src/main/java/com/esri/samples/open_mobile_scene_package/OpenMobileScenePackageSample.java:53]: Maybe this one got missed. It needs the `System.getProperty("data.dir")` parent directory argument.

**Review by tschie — APPROVED** [review]
Looks good. Just a couple things.

File: scene/open-mobile-scene-package/src/main/java/com/esri/samples/open_mobile_scene_package/OpenMobileScenePackageSample.java (hunk 2 above)
Comment (JonLavi) [reply scene/open-mobile-scene-package/src/main/java/com/esri/samples/open_mobile_scene_package/OpenMobileScenePackageSample.java:53]: I thought that's a change we're only doing in `dev` for now. 
Should I target this PR at `dev` instead?

Comment (tschie) [reply scene/open-mobile-scene-package/src/main/java/com/esri/samples/open_mobile_scene_package/OpenMobileScenePackageSample.java:53]: You're right. Thought it was targeting dev. Disregard.

**Review by Rachael-E — APPROVED** [review]
thanks @JonLavi looks good

**Comment by JonLavi** [comment]
Hi both,
I was trying to rebase and merge this into `dev`, so that we can verify against the sample viewer.
However I made a bit of a mess, so I'm closing this and merging a different PR with the exact changes (up until the botched rebase-attempt)
Sorry for the confusion!

**Pull request closed without merging**